test/integration/team-membership-integration.js

```diff
@@ -0,0 +1,34 @@
+export default function teamMembershipTests (t, organization) {
+  t.test('Gets teamMemberships', (t) => {
+    t.plan(3)
+    return organization.getTeamMemberships('7pIEx2fMx53SSR1jd7C46M')
+      .then((response) => {
+        t.ok(response.sys, 'sys')
+        t.ok(response.items, 'items')
+        t.ok(response.items[0].sys.type, 'TeamMembership')
+      })
+  })
+
+  t.test('Gets teamMembership', (t) => {
+    t.plan(3)
+    return organization.getTeamMembership('7pIEx2fMx53SSR1jd7C46M', '1tzDxN5zD2JbUjwzntAlLW')
+      .then((response) => {
+        t.ok(response.sys, 'sys')
+        t.ok(response.sys.id, '1tzDxN5zD2JbUjwzntAlLW')
+        t.ok(response.sys.type, 'TeamMembership')
+      })
+  })
+
+  t.test('Create and delete teamMembership', (t) => {
+    t.plan(3)
+    return organization.createTeamMembership('7pIEx2fMx53SSR1jd7C46M', {
+      admin: true, organizationMembershipId: '5mDCHGHvePI8NOzw0WaY2Z'
+    })
+      .then((teamMembership) => {
+        t.ok(teamMembership.sys, 'sys')
+        t.equal(teamMembership.admin, true)
+        t.ok(teamMembership.sys.type, 'TeamMembership')
+        teamMembership.delete()
```

Comment:
I think we need to wait on this async function and maybe t.ok it?

The way it's written now, if it fails, your tests will have some error outputs but I don't think the test will actually "fail", if that makes sense.

Reply:
hmm yes I see what you mean